### ISSUE TITLE
Bun CSS bundler no work make big crash

### ISSUE THREAD

**Issue by gotpop** [oven-sh/bun#14849]
### How can we reproduce the crash?

Bun no work make big crash 

### Relevant log output

```
styles: bun build --experimental-css ./index.css
============================================================
Bun v1.1.33 (247456b6) macOS Silicon
macOS v14.6.1
Args: "bun" "build" "--experimental-css" "./index.css"
Features: tsconfig tsconfig_paths 
Elapsed: 37ms | User: 3ms | Sys: 19ms
RSS: 17.09MB | Peak: 17.09MB | Commit: 1.07GB | Faults: 330

panic: Segmentation fault at address 0xAAAAAAAAAAAAAAA2
oh no: Bun has crashed. This indicates a bug in Bun, not your code.

To send a redacted crash report to Bun's team,
please file a GitHub issue using the link below:

 https://bun.report/1.1.33/Mb1247456bAhgggggC__uzw2F+ttiBmm/xJu7+pB+1ywJ__A2t1q1q1C91q1q1C

oh no: multiple threads are crashing
panic: TODO: we need to go deeper
 https://bun.report/1.1.33/Mb1247456bAhgggggCm//7Y291hXmz2/Zuo8+Ym8nmYml5xJu7+pB+1ywJ__A0eNoL8Xfxt1IoT1XIS01NUSjJV0jPV0hJTS1ILQIAbvcItw

[1]    87860 trace trap  bun build --experimental-css ./index.css
```

### Stack Trace (bun.report)

Bun v1.1.33 ([`247456b`](<https://github.com/oven-sh/bun/tree/247456b675d4bf4b9cd920e0a080b7002325c0df>)) on macos aarch64 [BuildCommand]

Segmentation fault at address 0xAAAAAAAAAAAAAAA2

- *2 unknown/js code*
- `src.resolver.resolve_path.relativePlatform__anon_96944`
- `src.css.css_parser.StyleSheet`
- `src.bundler.bundle_v2.ParseTask.getAST`
- `src.bundler.bundle_v2.ParseTask.callback`
- `Thread.PosixThreadImpl.spawn__anon_116617.Instance.entryFn`

Features: tsconfig, tsconfig\_paths, tsconfig\_paths

**Comment by nektro** [comment]
are you able to provide us with a copy of `index.css` ?

**Comment by noClaps** [comment]
Seems very similar to my comment here: https://github.com/oven-sh/bun/issues/14537#issuecomment-2439087636

**Comment by zackradisic** [comment]
Various CSS parser crashes have been fixed by #14832 

Feel free to reopen the issue if you still experience this crash!
